internal/plugins/ansible/v1/scaffolds/internal/templates/config/rbac/role.go

```diff
@@ -111,6 +111,8 @@ rules:
       - pods
       - pods/exec
       - pods/log
+      - serviceaccounts
+      - services
```

Comment:
What is the motivation for these additions?

Reply:
According to the users, both are required as well. We can test it.

Reply:
serviceaccounts and services are not required to fix the issue as they were already present. I think they're necessary for the Nginx sample, not directly related to the finalizers issue.